src/app/Mappings/components/MappingsActionsDropdown.tsx

```diff
@@ -85,14 +85,11 @@ const MappingsActionsDropdown: React.FunctionComponent<IMappingsActionsDropdownP
         toggleOpen={toggleDeleteModal}
         mutateFn={() => deleteMappingMutation.mutate(mapping)}
         mutateResult={deleteMappingMutation}
-        title={`Delete ${mappingType.toLowerCase()} mapping`}
+        title={`Delete ${mappingType.toLowerCase()} mapping?`}
```

Comment:
Let's emphasize that deleting this mapping has a lasting impact on user workflows by leading with "Permanently" to align with how we write deletion confirmation modals across other product areas. Then, we can rephrase the body copy slightly to make it a bit more personal to the user:

**Permanently delete network mapping?**
You will no longer be able to select network mapping "name" when you create a migration plan.